models/executor.js

```diff
@@ -5,4 +5,28 @@ module.exports = function(Executor) {
     }
     next();
   };
+
+  Executor.disableRemoteMethod('__delete__instances');
```

Comment:
Some of these models are managed and meant to be created/deleted only by PM or Mesh etc. They should only be queryable and not editable over the REST API.